h5netcdf/core.py

```diff
@@ -726,6 +747,7 @@ def __init__(self, path, mode="a", invalid_netcdf=None, phony_dims=None, **kwarg
             self._closed = False
 
         self._mode = mode
+        self._writable = not (mode == "r")
```

Comment:
Would this be the same as `mode != 'r'`?

Reply:
Now that you mention this...This looks much nicer. Why did I do this in the first place?